### PR TITLE
chore(rating): add a first batch of tests

### PR DESCRIPTION
**Pull request by Foxandxss**
Just a couple of tests, I will progressively add more.

Feel free to ask if you want to learn more about testing.

Also, removed sanity tests, not needed anymore.

This won't work yet because I have something to fix on the tooling. I will research that now with @robwormald

### CODE REVIEW

**Comment by Foxandxss** [comment]
Fixed that issue as well.

**Comment by Foxandxss** [comment]
Wait, broken.

**Comment by Foxandxss** [comment]
I think we should merge #58 first.

File: src/test/rating_test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+         expect(stars.length).toBe(10);
+       });
+     }));
+
+  it('should change the num of stars with `max`', injectAsync([TestComponentBuilder], (tcb) => {
+       const html = '<ngb-rating max="3"></ngb-rating>';
+
+       return tcb.overrideTemplate(TestComponent, html)
+           .createAsync(TestComponent)
+           .then((fixture) => {
```
Comment (wesleycho) [inline src/test/rating_test.ts]: How do you feel about `(fixture) =>` vs. `fixture =>`? This is a minor comment, all things considered.

**Comment by wesleycho** [comment]
Should we namespace the tests in the filenames, i.e. `rating.spec.ts` or `rating.test.ts` instead of `rating_test.ts`?

File: src/test/rating_test.ts (hunk 1 above)
Comment (Foxandxss) [inline src/test/rating_test.ts]: Personally I like the idea, but should we do that or stick to angular convention?

Comment (wesleycho) [inline src/test/rating_test.ts]: Is angular convention to always wrap it in parens?

Comment (Foxandxss) [inline src/test/rating_test.ts]: Yeah, they are doing that.

Comment (wesleycho) [inline src/test/rating_test.ts]: I'm fine with leaving that alone then - more of a code style comment than anything else.

**Comment by wesleycho** [comment]
Awesome, LGTM

**Pull request closed without merging**